neuralpredictors/layers/readouts/gaussian.py

```diff
@@ -277,13 +277,22 @@ def __init__(
         feature_reg_weight=None,
         gamma_readout=None,  # depricated, use feature_reg_weight instead
         return_weighted_features=False,
+        regularizer_type="l1",
+        gamma_sigma=0.1,
         **kwargs,
     ) -> None:
         super().__init__()
         self.feature_reg_weight = self.resolve_deprecated_gamma_readout(feature_reg_weight, gamma_readout, default=1.0)
         self.mean_activity = mean_activity
         # determines whether the Gaussian is isotropic or not
         self.gauss_type = gauss_type
+        self.regularizer_type = regularizer_type
+
+        if self.regularizer_type == "adaptive_log_norm":
+            self.gamma_sigma = gamma_sigma
+            self.adaptive = torch.nn.Parameter(torch.normal(mean=torch.ones(1, outdims), std=torch.ones(1, outdims)))
```

Comment:
@MaxFBurg these are not features, but additionally learnt regularisation coefficients (`self.features` are actually neuron embeddings)

we can rename it to `self.betas` as they are referred in the paper. Originally, there was a confusing with `gammas` called that way, thats why I wanted to give a more explicit name. Maybe `adaptive_beta` would be even better. 
What do you think ?

Reply:
I personally (and many others, too) do not like naming variable names after greek or latin characters (e.g. `beta`, `alpha`, `a`, `b`), because to a reader of the code that unfamiliar with the papers these names are not helpful. (I know we have `gamma` as variable name, and there were discussions that this is actually not a good variable name, but back then we kept it for backwards compatibility reasons). Therefore, in good coding practice variable names that convey their meaning are commonly used. So I like your idea of giving them an explicit name. What about `adaptive_neuron_reg_weight` or smth along these lines?

Reply:
There is also a section about that in the good research code handbook: https://goodresearch.dev/docs.html?highlight=raise#write-in-line-comments

Reply:
@MaxFBurg I will rename it to `adaptive_neuron_reg_coefs` then (imho, they are more like learns coefs then weights and weights might be confusing and the actual neurons weights are self.features)